reducer-todo/src/components/TodoList.js

```diff
@@ -1,9 +1,9 @@
-import React, { useReducer } from 'react';
+import React from 'react';
 
 const TodoList = ({ todoArray }) => {
     return (
         <div>
-            Todo List
+            <h2>Todo List via Redux</h2>
```

Comment:
You mapped over the todoArray, then you tried to return  todoArray.item you should be returning todo.item